src/wavm/memory.cpp

```diff
@@ -169,11 +170,18 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env, "__sbrk", I32, __sbrk, I32 increment)
     faabric::util::getLogger()->debug("S - sbrk - {}", increment);
 
     WAVMWasmModule* module = getExecutingWAVMModule();
+    I32 result;
     if (increment < 0) {
-        return module->shrinkMemory(-1 * increment);
+        PROF_START(sbrkShrink)
+        result = module->shrinkMemory(-1 * increment);
+        PROF_END(sbrkShrink)
     } else {
-        return module->growMemory(increment);
+        PROF_START(sbrkGrow)
+        result = module->growMemory(increment);
+        PROF_END(sbrkGrow)
     }
+
+    return result;
```

Comment:
The `PROF_START` and `PROF_END` here are useful to keep in in case this starts becoming a bottleneck (as all threads have to lock before changing memory.

Reply:
Ok. Just double-check the previous comment as I don't know if it applies or not.